app/mailers/contact_mailer.rb

```diff
@@ -0,0 +1,7 @@
+class ContactMailer < ApplicationMailer
+  def contact(email, message)
+    @email = email
+    @message = message
+    mail to: ENV['ADMIN_EMAIL'], from: @email, subject: I18n.t('contact_mailer.contact.subject')
```

Comment:
I'm not sure about the `from` field here. As it is now, this would send an email from that is from the `@email` value. This doesen't make much sense to me. Maybe the from address should be something like `'contact@targetmvd.com'` or similar

Reply:
Yeah `reply-to` would be better